diff --git a/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/batch/BatchModelingBot.java b/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/batch/BatchModelingBot.java
index b78df72..c36a275 100755
--- a/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/batch/BatchModelingBot.java
+++ b/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/batch/BatchModelingBot.java
@@ -32,7 +32,6 @@ import org.eclipse.emf.ecore.EReference;
 import org.eclipse.emf.ecore.EStructuralFeature;
 import org.eclipse.emf.ecore.resource.Resource;
 import org.eclipse.emf.ecore.util.EcoreUtil;
-import org.eclipse.emf.ecore.xmi.impl.XMIResourceImpl;
 import org.eclipse.emf.edit.command.AddCommand;
 import org.eclipse.emf.edit.command.RemoveCommand;
 import org.eclipse.emf.edit.command.SetCommand;
@@ -399,10 +398,9 @@ public class BatchModelingBot implements IModelingBot {
 
 	public void cancel(Processing processing) {
 		if (processing instanceof Wizard) {
-			int numberOfActionsToCancel = EEFModelingBotHelper.getNumberOfActionToUndo(processing);
+			int numberOfActionsToCancel = EEFModelingBotHelper.getNumberOfActionToCancel(processing);
 			for (int i = 0; i < numberOfActionsToCancel; i++) {
 				editingDomain.getCommandStack().undo();
-				;
 			}
 		} else {
 			editingDomain.getCommandStack().undo();
diff --git a/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/helper/EEFModelingBotHelper.java b/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/helper/EEFModelingBotHelper.java
index 8fe8d57..e028f36 100644
--- a/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/helper/EEFModelingBotHelper.java
+++ b/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/helper/EEFModelingBotHelper.java
@@ -1,6 +1,5 @@
 package org.eclipse.emf.eef.modelingBot.helper;
 
-import java.util.ArrayList;
 import java.util.Collection;
 
 import org.eclipse.emf.common.util.EList;
@@ -69,13 +68,26 @@ public class EEFModelingBotHelper {
 
 	public static int getNumberOfActionToUndo(Processing processing) {
 		EList<EObject> eContents = processing.eContents();
-		Collection<EObject> eContentsAction = new ArrayList<EObject>();
+		int actionstoUndo = 0;
 		for (EObject eObject : eContents) {
 			if (eObject instanceof EditAction && !(eObject instanceof Cancel)) {
-				eContentsAction.add(eObject);
+				actionstoUndo++;
+			} else if (eObject instanceof Wizard) {
+				actionstoUndo += getNumberOfActionToUndo((Processing) eObject);
 			}
 		}
-		return eContentsAction.size();
+		return actionstoUndo;
+	}
+	
+	public static int getNumberOfActionToCancel(Processing processing) {
+		EList<EObject> eContents = processing.eContents();
+		int actionstoCancel = 0;
+		for (EObject eObject : eContents) {
+			if (eObject instanceof EditAction && !(eObject instanceof Cancel)) {
+				actionstoCancel++;
+			}
+		}
+		return actionstoCancel;
 	}
 
 }
diff --git a/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/helper/PropertiesEditionHelper.java b/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/helper/PropertiesEditionHelper.java
index 37c54e6..62bb7a0 100755
--- a/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/helper/PropertiesEditionHelper.java
+++ b/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/helper/PropertiesEditionHelper.java
@@ -23,12 +23,9 @@ import org.eclipse.emf.ecore.EStructuralFeature;
 import org.eclipse.emf.edit.provider.IItemPropertySource;
 import org.eclipse.emf.eef.components.PropertiesEditionElement;
 import org.eclipse.emf.eef.extended.editor.ReferenceableObject;
-import org.eclipse.emf.eef.modelingBot.Action;
 import org.eclipse.emf.eef.modelingBot.SequenceType;
 import org.eclipse.emf.eef.modelingBot.Wizard;
 import org.eclipse.emf.eef.modelingBot.EEFActions.Add;
-import org.eclipse.emf.eef.modelingBot.EEFActions.Cancel;
-import org.eclipse.emf.eef.modelingBot.EEFActions.EditAction;
 import org.eclipse.emf.eef.modelingBot.interpreter.EEFInterpreter;
 import org.eclipse.emf.eef.modelingBot.swtbot.SWTEEFBot;
 import org.eclipse.emf.eef.modelingBot.ui.utils.WrappedSWTBotRadio;
@@ -252,6 +249,13 @@ public class PropertiesEditionHelper {
 		String label = ((ElementEditor) propertiesEditionElement.getViews()
 				.get(0)).getQualifiedIdentifier();
 		bot.browseButtonFlatReferencesTable(label).click();
+		final SWTBotTable table = bot.table(1);
+		int rowCount = table.rowCount();
+		for (int i = 0; i < rowCount; i++) {
+			SWTBotButton buttonRemove = bot.button(1);
+			buttonRemove.click();
+			SWTBotHelper.waitAllUiEvents();
+		}
 		bot.selectInActiveTable(values);
 		SWTBotButton buttonAdd = bot.button(0);
 		buttonAdd.click();
diff --git a/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/swtbot/SWTEEFBot.java b/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/swtbot/SWTEEFBot.java
index 0f60bfb..af99600 100755
--- a/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/swtbot/SWTEEFBot.java
+++ b/plugins/org.eclipse.emf.eef.modelingbot/src/org/eclipse/emf/eef/modelingBot/swtbot/SWTEEFBot.java
@@ -359,14 +359,15 @@ public class SWTEEFBot extends SWTWorkbenchBot implements IModelingBot {
 		container = getEObjectFromReferenceableEObject(referenceableObjectContainer);
 		assertNotNull("No container is found to launch wizard.", container);
 		SWTBotHelper.waitAllUiEvents();
-		final SWTBotTreeItem selectNode = selectNode(editor, container);
-		assertNotNull("No element is selected in the editor", selectNode);
-		if (sequenceType.equals(SequenceType.PROPERTIES_VIEW))
+		assertFalse("The set action must be define in a sequence.", sequenceType == null);
+		if (sequenceType.equals(SequenceType.PROPERTIES_VIEW)) {
+			final SWTBotTreeItem selectNode = selectNode(editor, container);
+			assertNotNull("No element is selected in the editor", selectNode);
 			initPropertiesViewTab(propertiesEditionElement, selectNode);
-		else
+		} else if (sequenceType.equals(SequenceType.WIZARD)) {
 			initTab(propertiesEditionElement);
-		assertFalse("The set action must be define in a sequence.", sequenceType == null);
-		propertiesEdition.addFeature(selectNode, propertiesEditionElement, referenceableObject, sequenceType);
+		}
+		propertiesEdition.addFeature(null, propertiesEditionElement, referenceableObject, sequenceType);
 		SWTBotHelper.waitAllUiEvents();
 		return (EObject)EEFModelHelper.eGet(container, propertiesEditionElement.getModel());
 	}
diff --git a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/addTableComposition.modelingbot b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/addTableComposition.modelingbot
index 48c573c..43be284 100644
--- a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/addTableComposition.modelingbot
+++ b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/addTableComposition.modelingbot
@@ -8,11 +8,22 @@
     </processings>
     <processings xsi:type="eef-test_1:Check"/>
     <processings xsi:type="eef-mbot:DetailsPage">
-      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.1">
-        <processings xsi:type="eef-test_1:Add" name="table compo sample" documentation="" referenceableObject="//@sequences.0/@processings.1">
-          <propertiesEditionElement href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LAwwvEd-uP4lw9e0IMw"/>
-          <type href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TableCompositionEditorSample"/>
-          <eContainingFeature xsi:type="ecore:EReference" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root/samples"/>
+      <processings xsi:type="eef-test_1:Add" name="table compo sample" documentation="" referenceableObject="//@sequences.0/@processings.1">
+        <type href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TableCompositionEditorSample"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root/samples"/>
+      </processings>
+      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.3/@processings.0">
+        <processings xsi:type="eef-mbot:Wizard">
+          <processings xsi:type="eef-test_1:Add" name="sample" referenceableObject="//@sequences.0/@processings.3/@processings.0">
+            <propertiesEditionElement href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LNgwvEd-uP4lw9e0IMw"/>
+            <type href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Sample"/>
+            <eContainingFeature xsi:type="ecore:EReference" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TableCompositionEditorSample/tablecompositionRequiredProperty"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="text required property" referenceableObject="//@sequences.0/@processings.3/@processings.1/@processings.0/@processings.0">
+            <propertiesEditionElement href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_CDVckQ_gEd-fwJDNg0onkA"/>
+            <values>sample</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Sample/textRequiredProperty"/>
+          </processings>
         </processings>
       </processings>
       <processings xsi:type="eef-test:Save"/>
diff --git a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/cancelAddTableComposition.modelingbot b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/cancelAddTableComposition.modelingbot
index 5f6528e..e9f2edd 100644
--- a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/cancelAddTableComposition.modelingbot
+++ b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/cancelAddTableComposition.modelingbot
@@ -1,22 +1,36 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<eef-mbot:ModelingBot xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" xmlns:eef-mbot="http://www.eclipse.org/emf/eef/mbot/1.0.0" xmlns:eef-test="http://www.eclipse.org/emf/eef/test/EclipseActions/1.0.0" xmlns:eef-test_1="http://www.eclipse.org/emf/eef/test/EEFActions/1.0.0">
-  <propertiesEditionContext href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LAAwvEd-uP4lw9e0IMw"/>
-  <sequences xsi:type="eef-mbot:Scenario">
-    <processings xsi:type="eef-test:CreateProject" name="addTableComposition" projectName="addTableComposition"/>
-    <processings xsi:type="eef-test:CreateModel" name="eefnr" documentation="" modelName="addTableComposition.eefnr" path="addTableComposition">
-      <root href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root"/>
-    </processings>
-    <processings xsi:type="eef-test_1:Check"/>
-    <processings xsi:type="eef-mbot:DetailsPage">
-      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.1">
-        <processings xsi:type="eef-test_1:Add" name="table compo sample" documentation="" referenceableObject="//@sequences.0/@processings.1">
-          <type href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TableCompositionEditorSample"/>
-          <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root/samples"/>
-        </processings>
-      </processings>
-      <processings xsi:type="eef-test:Save"/>
-    </processings>
-    <processings xsi:type="eef-test_1:Check"/>
-    <processings xsi:type="eef-test:RemoveProject" name="addTableComposition" projectName="addTableComposition"/>
-  </sequences>
-</eef-mbot:ModelingBot>
+<?xml version="1.0" encoding="UTF-8"?>
+<eef-mbot:ModelingBot xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" xmlns:eef-mbot="http://www.eclipse.org/emf/eef/mbot/1.0.0" xmlns:eef-test="http://www.eclipse.org/emf/eef/test/EclipseActions/1.0.0" xmlns:eef-test_1="http://www.eclipse.org/emf/eef/test/EEFActions/1.0.0">
+  <propertiesEditionContext href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LAAwvEd-uP4lw9e0IMw"/>
+  <sequences xsi:type="eef-mbot:Scenario">
+    <processings xsi:type="eef-test:CreateProject" name="addTableComposition" projectName="addTableComposition"/>
+    <processings xsi:type="eef-test:CreateModel" name="eefnr" documentation="" modelName="addTableComposition.eefnr" path="addTableComposition">
+      <root href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root"/>
+    </processings>
+    <processings xsi:type="eef-test_1:Check"/>
+    <processings xsi:type="eef-mbot:DetailsPage">
+      <processings xsi:type="eef-test_1:Add" name="table compo sample" documentation="" referenceableObject="//@sequences.0/@processings.1">
+        <type href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TableCompositionEditorSample"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root/samples"/>
+      </processings>
+      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.3/@processings.0">
+        <processings xsi:type="eef-mbot:Wizard">
+          <processings xsi:type="eef-test_1:Add" name="sample" referenceableObject="//@sequences.0/@processings.3/@processings.0">
+            <propertiesEditionElement href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LNgwvEd-uP4lw9e0IMw"/>
+            <type href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Sample"/>
+            <eContainingFeature xsi:type="ecore:EReference" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TableCompositionEditorSample/tablecompositionRequiredProperty"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="text required property" referenceableObject="//@sequences.0/@processings.3/@processings.1/@processings.0/@processings.0">
+            <propertiesEditionElement href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_CDVckQ_gEd-fwJDNg0onkA"/>
+            <values>sample</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Sample/textRequiredProperty"/>
+          </processings>
+          <processings xsi:type="eef-test_1:Cancel" processing="//@sequences.0/@processings.3/@processings.1/@processings.0"/>
+        </processings>
+        <processings xsi:type="eef-test_1:Cancel" processing="//@sequences.0/@processings.3/@processings.1"/>
+      </processings>
+      <processings xsi:type="eef-test:Save"/>
+    </processings>
+    <processings xsi:type="eef-test_1:Check"/>
+    <processings xsi:type="eef-test:RemoveProject" name="addTableComposition" projectName="addTableComposition"/>
+  </sequences>
+</eef-mbot:ModelingBot>
diff --git a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/editSetReferenceRefTable.modelingbot b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/editSetReferenceRefTable.modelingbot
deleted file mode 100644
index e938266..0000000
--- a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/editSetReferenceRefTable.modelingbot
+++ /dev/null
@@ -1,38 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<eef-mbot:ModelingBot xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" xmlns:eef-mbot="http://www.eclipse.org/emf/eef/mbot/1.0.0" xmlns:eef-test="http://www.eclipse.org/emf/eef/test/EclipseActions/1.0.0" xmlns:eef-test_1="http://www.eclipse.org/emf/eef/test/EEFActions/1.0.0">
-  <propertiesEditionContext href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LAAwvEd-uP4lw9e0IMw"/>
-  <sequences xsi:type="eef-mbot:Scenario">
-    <processings xsi:type="eef-test:CreateProject" name="setReferenceRefTable" projectName="setReferenceRefTable"/>
-    <processings xsi:type="eef-test:CreateModel" name="eefnr" documentation="" modelName="setReferenceRefTable.eefnr" path="setReferenceRefTable">
-      <root href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root"/>
-    </processings>
-    <processings xsi:type="eef-test_1:Check"/>
-    <processings xsi:type="eef-mbot:DetailsPage">
-      <processings xsi:type="eef-test_1:Add" name="ref table sample" referenceableObject="//@sequences.0/@processings.1">
-        <type href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//ReferencesTableSample"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root/samples"/>
-      </processings>
-      <processings xsi:type="eef-test_1:Add" name="total sample 1" referenceableObject="//@sequences.0/@processings.1">
-        <type href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TotalSample"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root/samples"/>
-      </processings>
-      <processings xsi:type="eef-test_1:Add" name="total sample 2" referenceableObject="//@sequences.0/@processings.1">
-        <type href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TotalSample"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root/samples"/>
-      </processings>
-      <processings xsi:type="eef-test_1:SetReference" name="required property" referenceableObject="//@sequences.0/@processings.3/@processings.0" values="//@sequences.0/@processings.3/@processings.1">
-        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LLQwvEd-uP4lw9e0IMw"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//ReferencesTableSample/referencestableRequiredProperty"/>
-      </processings>
-      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.3/@processings.0">
-        <processings xsi:type="eef-test_1:SetReference" name="required property" referenceableObject="//@sequences.0/@processings.3/@processings.0" values="//@sequences.0/@processings.3/@processings.2">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LLQwvEd-uP4lw9e0IMw"/>
-          <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//ReferencesTableSample/referencestableRequiredProperty"/>
-        </processings>
-      </processings>
-      <processings xsi:type="eef-test:Save"/>
-    </processings>
-    <processings xsi:type="eef-test_1:Check"/>
-    <processings xsi:type="eef-test:RemoveProject" name="setReferenceRefTable" projectName="setReferenceRefTable"/>
-  </sequences>
-</eef-mbot:ModelingBot>
diff --git a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/undoAddTableComposition.modelingbot b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/undoAddTableComposition.modelingbot
index 4a02cde..2f4188f 100644
--- a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/undoAddTableComposition.modelingbot
+++ b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/undoAddTableComposition.modelingbot
@@ -1,22 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <eef-mbot:ModelingBot xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" xmlns:eef-mbot="http://www.eclipse.org/emf/eef/mbot/1.0.0" xmlns:eef-test="http://www.eclipse.org/emf/eef/test/EclipseActions/1.0.0" xmlns:eef-test_1="http://www.eclipse.org/emf/eef/test/EEFActions/1.0.0">
-  <propertiesEditionContext href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LAAwvEd-uP4lw9e0IMw"/>
+  <propertiesEditionContext href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LAAwvEd-uP4lw9e0IMw"/>
   <sequences xsi:type="eef-mbot:Scenario">
     <processings xsi:type="eef-test:CreateProject" name="undoAddTableComposition" projectName="undoAddTableComposition"/>
-    <processings xsi:type="eef-test:CreateModel" name="eefnr" documentation="" modelName="addTableComposition.eefnr" path="addTableComposition">
-      <root href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root"/>
+    <processings xsi:type="eef-test:CreateModel" name="eefnr" documentation="" modelName="undoAddTableComposition.eefnr" path="undoAddTableComposition">
+      <root href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root"/>
     </processings>
     <processings xsi:type="eef-test_1:Check"/>
     <processings xsi:type="eef-mbot:DetailsPage">
-      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.1">
-        <processings xsi:type="eef-test_1:Add" name="table compo sample" documentation="" referenceableObject="//@sequences.0/@processings.1">
-          <type href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TableCompositionEditorSample"/>
-          <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root/samples"/>
+      <processings xsi:type="eef-test_1:Add" name="table compo sample" documentation="" referenceableObject="//@sequences.0/@processings.1">
+        <type href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TableCompositionEditorSample"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Root/samples"/>
+      </processings>
+      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.3/@processings.0">
+        <processings xsi:type="eef-mbot:Wizard">
+          <processings xsi:type="eef-test_1:Add" name="sample" referenceableObject="//@sequences.0/@processings.3/@processings.0">
+            <propertiesEditionElement href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LNgwvEd-uP4lw9e0IMw"/>
+            <type href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Sample"/>
+            <eContainingFeature xsi:type="ecore:EReference" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TableCompositionEditorSample/tablecompositionRequiredProperty"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="text required property" referenceableObject="//@sequences.0/@processings.3/@processings.1/@processings.0/@processings.0">
+            <propertiesEditionElement href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_CDVckQ_gEd-fwJDNg0onkA"/>
+            <values>sample</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="../../../../plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//Sample/textRequiredProperty"/>
+          </processings>
         </processings>
       </processings>
       <processings xsi:type="eef-test:Save"/>
       <processings xsi:type="eef-test_1:Check"/>
       <processings xsi:type="eef-test:Undo"/>
+      <processings xsi:type="eef-test:Save"/>
     </processings>
     <processings xsi:type="eef-test_1:Check"/>
     <processings xsi:type="eef-test:RemoveProject" name="undoAddTableComposition" projectName="undoAddTableComposition"/>
diff --git a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/unsetEOFCV.modelingbot b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/unsetEOFCV.modelingbot
index 24b7350..e8ca93c 100644
--- a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/unsetEOFCV.modelingbot
+++ b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/unsetEOFCV.modelingbot
@@ -22,7 +22,7 @@
       </processings>
       <processings xsi:type="eef-test_1:SetReference" name="required property" referenceableObject="//@sequences.0/@processings.3/@processings.0" values="//@sequences.0/@processings.3/@processings.1">
         <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LKgwvEd-uP4lw9e0IMw"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TotalSample/eobjectflatcomboviewerRequiredProperty"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//EObjectFlatComboViewerSample/eobjectflatcomboviewerRequiredPropery"/>
       </processings>
       <processings xsi:type="eef-test_1:SetReference" name="optional property" referenceableObject="//@sequences.0/@processings.3/@processings.0" values="//@sequences.0/@processings.3/@processings.2">
         <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LKwwvEd-uP4lw9e0IMw"/>
diff --git a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/unsetReferenceRefTable.modelingbot b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/unsetReferenceRefTable.modelingbot
index 7937fc0..3e6cd96 100644
--- a/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/unsetReferenceRefTable.modelingbot
+++ b/tests/org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/unsetReferenceRefTable.modelingbot
@@ -31,7 +31,7 @@
       <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.3/@processings.0">
         <processings xsi:type="eef-test_1:UnsetReference" name="optional property" referenceableObject="//@sequences.0/@processings.3/@processings.0" values="//@sequences.0/@processings.3/@processings.2">
           <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg.edit/models/eefnr.components#_Fx_LLgwvEd-uP4lw9e0IMw"/>
-          <feature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//TotalSample/referencestableOptionalProperty"/>
+          <feature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.eef.tests.nonreg/model/eefnr.ecore#//ReferencesTableSample/referencestableOptionalProperty"/>
         </processings>
       </processings>
       <processings xsi:type="eef-test:Save"/>
diff --git a/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/AllTests.java b/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/AllTests.java
index d602af1..dd05f88 100644
--- a/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/AllTests.java
+++ b/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/AllTests.java
@@ -73,7 +73,6 @@ import org.eclipse.emf.eef.tests.nonreg.wizard.WizardMBotCancelSetReferenceFlatR
 import org.eclipse.emf.eef.tests.nonreg.wizard.WizardMBotCancelSetReferenceRefTable;
 import org.eclipse.emf.eef.tests.nonreg.wizard.WizardMBotEditSetReferenceAEOFCV;
 import org.eclipse.emf.eef.tests.nonreg.wizard.WizardMBotEditSetReferenceFlatRefTable;
-import org.eclipse.emf.eef.tests.nonreg.wizard.WizardMBotEditSetReferenceRefTable;
 import org.eclipse.emf.eef.tests.nonreg.wizard.WizardMBotRedoAddTableComposition;
 import org.eclipse.emf.eef.tests.nonreg.wizard.WizardMBotRedoSetAttributeRadio;
 import org.eclipse.emf.eef.tests.nonreg.wizard.WizardMBotRedoSetReferenceAEOFCV;
@@ -149,7 +148,6 @@ public class AllTests extends TestCase {
 		suite.addTestSuite(WizardMBotCancelSetReferenceRefTable.class);
 		suite.addTestSuite(WizardMBotEditSetReferenceAEOFCV.class);
 		suite.addTestSuite(WizardMBotEditSetReferenceFlatRefTable.class);
-		suite.addTestSuite(WizardMBotEditSetReferenceRefTable.class);
 		suite.addTestSuite(WizardMBotRedoAddTableComposition.class);
 		suite.addTestSuite(WizardMBotRedoSetAttributeRadio.class);
 		suite.addTestSuite(WizardMBotRedoSetReferenceAEOFCV.class);
diff --git a/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/wizard/WizardMBotCancelAddTableComposition.java b/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/wizard/WizardMBotCancelAddTableComposition.java
index 9808d03..4d7f1d3 100644
--- a/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/wizard/WizardMBotCancelAddTableComposition.java
+++ b/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/wizard/WizardMBotCancelAddTableComposition.java
@@ -16,7 +16,7 @@ package org.eclipse.emf.eef.tests.nonreg.wizard;
 public class WizardMBotCancelAddTableComposition extends org.eclipse.emf.eef.modelingBot.testcase.AbstractComposedModelingBotTestCase {
 
 	public void testModelingBot() throws Exception {
-		bot.runModelingBot("org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/addTableComposition.modelingbot");
+		bot.runModelingBot("org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/cancelAddTableComposition.modelingbot");
 	}
 
 }
diff --git a/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/wizard/WizardMBotEditSetReferenceRefTable.java b/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/wizard/WizardMBotEditSetReferenceRefTable.java
deleted file mode 100644
index 0ae94de..0000000
--- a/tests/org.eclipse.emf.eef.tests.nonreg.junit/src-mbot/org/eclipse/emf/eef/tests/nonreg/wizard/WizardMBotEditSetReferenceRefTable.java
+++ /dev/null
@@ -1,22 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2011, 2012 Obeo.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *     Obeo - initial API and implementation
- *******************************************************************************/
-package org.eclipse.emf.eef.tests.nonreg.wizard;
-
-/**
- * @author <a href="mailto:riadh.hajamor@obeo.fr">Riadh HAJ AMOR</a>
- */
-public class WizardMBotEditSetReferenceRefTable extends org.eclipse.emf.eef.modelingBot.testcase.AbstractComposedModelingBotTestCase {
-
-	public void testModelingBot() throws Exception {
-		bot.runModelingBot("org.eclipse.emf.eef.tests.nonreg.junit/models/wizard_nonreg/setReferenceRefTable.modelingbot");
-	}
-
-}
diff --git a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/addAdvancedTableComposition.modelingbot b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/addAdvancedTableComposition.modelingbot
index c89a6f6..1cc6648 100644
--- a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/addAdvancedTableComposition.modelingbot
+++ b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/addAdvancedTableComposition.modelingbot
@@ -19,20 +19,22 @@
         <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/place"/>
       </processings>
       <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.1">
-        <processings xsi:type="eef-test_1:Add" name="site1" documentation="" referenceableObject="//@sequences.0/@processings.1">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_P9i7gJktEd6sxtwdpecy3w"/>
-          <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site"/>
-          <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/sites"/>
-        </processings>
-        <processings xsi:type="eef-test_1:SetAttribute" name="name" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBPJkmEd6sxtwdpecy3w"/>
-          <values>site name</values>
-          <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/name"/>
-        </processings>
-        <processings xsi:type="eef-test_1:SetAttribute" name="documentation" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBO5kmEd6sxtwdpecy3w"/>
-          <values>this is a comment</values>
-          <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/documentation"/>
+        <processings xsi:type="eef-mbot:Wizard">
+          <processings xsi:type="eef-test_1:Add" name="site1" documentation="" referenceableObject="//@sequences.0/@processings.1">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_P9i7gJktEd6sxtwdpecy3w"/>
+            <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site"/>
+            <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/sites"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="name" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0/@processings.0">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBPJkmEd6sxtwdpecy3w"/>
+            <values>site name</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/name"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="documentation" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0/@processings.0">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBO5kmEd6sxtwdpecy3w"/>
+            <values>this is a comment</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/documentation"/>
+          </processings>
         </processings>
       </processings>
       <processings xsi:type="eef-test:Save"/>
diff --git a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/cancelAddAdvancedTableComposition.modelingbot b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/cancelAddAdvancedTableComposition.modelingbot
index 9924003..5c02196 100644
--- a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/cancelAddAdvancedTableComposition.modelingbot
+++ b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/cancelAddAdvancedTableComposition.modelingbot
@@ -19,22 +19,25 @@
         <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/place"/>
       </processings>
       <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.1">
-        <processings xsi:type="eef-test_1:Add" name="site1" documentation="" referenceableObject="//@sequences.0/@processings.1">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_P9i7gJktEd6sxtwdpecy3w"/>
-          <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site"/>
-          <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/sites"/>
+        <processings xsi:type="eef-mbot:Wizard">
+          <processings xsi:type="eef-test_1:Add" name="site1" documentation="" referenceableObject="//@sequences.0/@processings.1">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_P9i7gJktEd6sxtwdpecy3w"/>
+            <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site"/>
+            <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/sites"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="name" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0/@processings.0">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBPJkmEd6sxtwdpecy3w"/>
+            <values>site name</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/name"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="documentation" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0/@processings.0">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBO5kmEd6sxtwdpecy3w"/>
+            <values>this is a comment</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/documentation"/>
+          </processings>
+          <processings xsi:type="eef-test_1:Cancel" processing="//@sequences.0/@processings.3/@processings.2/@processings.0"/>
         </processings>
-        <processings xsi:type="eef-test_1:SetAttribute" name="name" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBPJkmEd6sxtwdpecy3w"/>
-          <values>site name</values>
-          <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/name"/>
-        </processings>
-        <processings xsi:type="eef-test_1:SetAttribute" name="documentation" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBO5kmEd6sxtwdpecy3w"/>
-          <values>this is a comment</values>
-          <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/documentation"/>
-        </processings>
-        <processings xsi:type="eef-test_1:Cancel"/>
+        <processings xsi:type="eef-test_1:Cancel" processing="//@sequences.0/@processings.3/@processings.2"/>
       </processings>
       <processings xsi:type="eef-test:Save"/>
     </processings>
diff --git a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/redoAddAdvTableCompo.modelingbot b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/redoAddAdvTableCompo.modelingbot
index 624e327..4a151a9 100644
--- a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/redoAddAdvTableCompo.modelingbot
+++ b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/redoAddAdvTableCompo.modelingbot
@@ -8,32 +8,42 @@
     </processings>
     <processings xsi:type="eef-test_1:Check"/>
     <processings xsi:type="eef-mbot:DetailsPage">
-      <processings xsi:type="eef-test_1:Add" name="site1" documentation="" referenceableObject="//@sequences.0/@processings.1">
-        <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/sites"/>
+      <processings xsi:type="eef-test_1:SetAttribute" name="name" documentation="" referenceableObject="//@sequences.0/@processings.1">
+        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_0lme4Bi5EeCOk6BuQTSdWA"/>
+        <values>conf name</values>
+        <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/name"/>
       </processings>
-      <processings xsi:type="eef-mbot:Wizard">
-        <processings xsi:type="eef-test_1:Add" name="room1" referenceableObject="//@sequences.0/@processings.3/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_r1ZxAG3eEeC0ELVFaUipLQ"/>
-          <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Room"/>
-          <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/rooms"/>
-        </processings>
-        <processings xsi:type="eef-test_1:SetAttribute" name="name" referenceableObject="//@sequences.0/@processings.3/@processings.1/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_O_1wYG3dEeC0ELVFaUipLQ"/>
-          <values>room1</values>
-          <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Room/name"/>
-        </processings>
-        <processings xsi:type="eef-test_1:SetAttribute" name="capacity" referenceableObject="//@sequences.0/@processings.3/@processings.1/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_SM3tEG3dEeC0ELVFaUipLQ"/>
-          <values>35</values>
-          <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Room/capacity"/>
+      <processings xsi:type="eef-test_1:SetAttribute" name="place" referenceableObject="//@sequences.0/@processings.1">
+        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_OJpU8JktEd6sxtwdpecy3w"/>
+        <values>place name</values>
+        <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/place"/>
+      </processings>
+      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.1">
+        <processings xsi:type="eef-mbot:Wizard">
+          <processings xsi:type="eef-test_1:Add" name="site1" documentation="" referenceableObject="//@sequences.0/@processings.1">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_P9i7gJktEd6sxtwdpecy3w"/>
+            <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site"/>
+            <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/sites"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="name" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0/@processings.0">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBPJkmEd6sxtwdpecy3w"/>
+            <values>site name</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/name"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="documentation" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0/@processings.0">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBO5kmEd6sxtwdpecy3w"/>
+            <values>this is a comment</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/documentation"/>
+          </processings>
         </processings>
       </processings>
       <processings xsi:type="eef-test:Save"/>
-      <processings xsi:type="eef-test:Undo" name="add room1"/>
+      <processings xsi:type="eef-test_1:Check"/>
+      <processings xsi:type="eef-test:Undo"/>
       <processings xsi:type="eef-test:Save"/>
       <processings xsi:type="eef-test_1:Check"/>
-      <processings xsi:type="eef-test:Redo" name="add room1" documentation=""/>
+      <processings xsi:type="eef-test:Redo"/>
+      <processings xsi:type="eef-test:Save"/>
     </processings>
     <processings xsi:type="eef-test_1:Check"/>
     <processings xsi:type="eef-test:RemoveProject" name="redoAddAdvTableCompo" projectName="redoAddAdvTableCompo"/>
diff --git a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/undoAddAdvTableCompo.modelingbot b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/undoAddAdvTableCompo.modelingbot
index 4621e9c..a083182 100644
--- a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/undoAddAdvTableCompo.modelingbot
+++ b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/undoAddAdvTableCompo.modelingbot
@@ -8,29 +8,39 @@
     </processings>
     <processings xsi:type="eef-test_1:Check"/>
     <processings xsi:type="eef-mbot:DetailsPage">
-      <processings xsi:type="eef-test_1:Add" name="site1" documentation="" referenceableObject="//@sequences.0/@processings.1">
-        <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/sites"/>
+      <processings xsi:type="eef-test_1:SetAttribute" name="name" documentation="" referenceableObject="//@sequences.0/@processings.1">
+        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_0lme4Bi5EeCOk6BuQTSdWA"/>
+        <values>conf name</values>
+        <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/name"/>
       </processings>
-      <processings xsi:type="eef-mbot:Wizard">
-        <processings xsi:type="eef-test_1:Add" name="room1" referenceableObject="//@sequences.0/@processings.3/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_r1ZxAG3eEeC0ELVFaUipLQ"/>
-          <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Room"/>
-          <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/rooms"/>
-        </processings>
-        <processings xsi:type="eef-test_1:SetAttribute" name="name" referenceableObject="//@sequences.0/@processings.3/@processings.1/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_O_1wYG3dEeC0ELVFaUipLQ"/>
-          <values>room1</values>
-          <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Room/name"/>
-        </processings>
-        <processings xsi:type="eef-test_1:SetAttribute" name="capacity" referenceableObject="//@sequences.0/@processings.3/@processings.1/@processings.0">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_SM3tEG3dEeC0ELVFaUipLQ"/>
-          <values>35</values>
-          <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Room/capacity"/>
+      <processings xsi:type="eef-test_1:SetAttribute" name="place" referenceableObject="//@sequences.0/@processings.1">
+        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_OJpU8JktEd6sxtwdpecy3w"/>
+        <values>place name</values>
+        <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/place"/>
+      </processings>
+      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.1">
+        <processings xsi:type="eef-mbot:Wizard">
+          <processings xsi:type="eef-test_1:Add" name="site1" documentation="" referenceableObject="//@sequences.0/@processings.1">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_P9i7gJktEd6sxtwdpecy3w"/>
+            <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site"/>
+            <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/sites"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="name" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0/@processings.0">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBPJkmEd6sxtwdpecy3w"/>
+            <values>site name</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/name"/>
+          </processings>
+          <processings xsi:type="eef-test_1:SetAttribute" name="documentation" referenceableObject="//@sequences.0/@processings.3/@processings.2/@processings.0/@processings.0">
+            <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBO5kmEd6sxtwdpecy3w"/>
+            <values>this is a comment</values>
+            <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Site/documentation"/>
+          </processings>
         </processings>
       </processings>
       <processings xsi:type="eef-test:Save"/>
-      <processings xsi:type="eef-test:Undo" name="add room1"/>
+      <processings xsi:type="eef-test_1:Check"/>
+      <processings xsi:type="eef-test:Undo"/>
+      <processings xsi:type="eef-test:Save"/>
     </processings>
     <processings xsi:type="eef-test_1:Check"/>
     <processings xsi:type="eef-test:RemoveProject" name="undoAddAdvTableCompo" projectName="undoAddAdvTableCompo"/>
diff --git a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/unsetReferenceEOFCV.modelingbot b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/unsetReferenceEOFCV.modelingbot
index 84b9729..c1b979a 100644
--- a/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/unsetReferenceEOFCV.modelingbot
+++ b/tests/org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/unsetReferenceEOFCV.modelingbot
@@ -1,60 +1,67 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <eef-mbot:ModelingBot xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" xmlns:eef-mbot="http://www.eclipse.org/emf/eef/mbot/1.0.0" xmlns:eef-test="http://www.eclipse.org/emf/eef/test/EclipseActions/1.0.0" xmlns:eef-test_1="http://www.eclipse.org/emf/eef/test/EEFActions/1.0.0">
-  <propertiesEditionContext href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvIaEJkmEd6sxtwdpecy3w"/>
+  <propertiesEditionContext href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvIaEJkmEd6sxtwdpecy3w"/>
   <sequences xsi:type="eef-mbot:Scenario">
     <processings xsi:type="eef-test:CreateProject" name="unsetReferenceEOFCV" projectName="unsetReferenceEOFCV"/>
     <processings xsi:type="eef-test:CreateModel" name="conference" documentation="" modelName="unsetReferenceEOFCV.conference" path="unsetReferenceEOFCV">
-      <root href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference"/>
+      <root href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference"/>
     </processings>
     <processings xsi:type="eef-test_1:Check"/>
     <processings xsi:type="eef-mbot:DetailsPage">
       <processings xsi:type="eef-test_1:Add" name="topic1" documentation="" referenceableObject="//@sequences.0/@processings.1">
-        <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Topic"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/topics"/>
+        <type href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Topic"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/topics"/>
       </processings>
       <processings xsi:type="eef-test_1:SetAttribute" name="description" referenceableObject="//@sequences.0/@processings.3/@processings.0">
-        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBN5kmEd6sxtwdpecy3w"/>
+        <propertiesEditionElement href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBN5kmEd6sxtwdpecy3w"/>
         <values>topic1</values>
-        <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Topic/description"/>
+        <eContainingFeature xsi:type="ecore:EAttribute" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Topic/description"/>
+      </processings>
+      <processings xsi:type="eef-test_1:SetAttribute" name="documentation" referenceableObject="//@sequences.0/@processings.3/@processings.0">
+        <propertiesEditionElement href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBOZkmEd6sxtwdpecy3w"/>
+        <values>documentation</values>
+        <eContainingFeature xsi:type="ecore:EAttribute" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Topic/documentation"/>
       </processings>
       <processings xsi:type="eef-test_1:Add" name="presenter1" referenceableObject="//@sequences.0/@processings.1">
-        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_-S1ZMBi5EeCOk6BuQTSdWA"/>
-        <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Person"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/participants"/>
+        <type href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Person"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/participants"/>
       </processings>
-      <processings xsi:type="eef-test_1:SetAttribute" name="firstname" referenceableObject="//@sequences.0/@processings.3/@processings.2">
-        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBKJkmEd6sxtwdpecy3w"/>
+      <processings xsi:type="eef-test_1:SetAttribute" name="firstname" referenceableObject="//@sequences.0/@processings.3/@processings.3">
+        <propertiesEditionElement href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBKJkmEd6sxtwdpecy3w"/>
         <values>topic presenter</values>
-        <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Person/firstname"/>
+        <eContainingFeature xsi:type="ecore:EAttribute" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Person/firstname"/>
       </processings>
-    </processings>
-    <processings xsi:type="eef-mbot:DetailsPage">
       <processings xsi:type="eef-test_1:Add" name="talk1" referenceableObject="//@sequences.0/@processings.1">
-        <type href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/talks"/>
+        <type href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Conference/talks"/>
       </processings>
-      <processings xsi:type="eef-test_1:SetReference" name="topic" referenceableObject="//@sequences.0/@processings.4/@processings.0" values="//@sequences.0/@processings.3/@processings.0">
-        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBMZkmEd6sxtwdpecy3w"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/topic"/>
+      <processings xsi:type="eef-test_1:SetReference" name="topic" referenceableObject="//@sequences.0/@processings.3/@processings.5" values="//@sequences.0/@processings.3/@processings.0">
+        <propertiesEditionElement href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBMZkmEd6sxtwdpecy3w"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/topic"/>
       </processings>
-      <processings xsi:type="eef-test_1:SetAttribute" name="title" referenceableObject="//@sequences.0/@processings.4/@processings.0">
-        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBMJkmEd6sxtwdpecy3w"/>
+      <processings xsi:type="eef-test_1:SetAttribute" name="title" referenceableObject="//@sequences.0/@processings.3/@processings.5">
+        <propertiesEditionElement href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBMJkmEd6sxtwdpecy3w"/>
         <values>talk name</values>
-        <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/title"/>
+        <eContainingFeature xsi:type="ecore:EAttribute" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/title"/>
       </processings>
-      <processings xsi:type="eef-test_1:SetAttribute" name="documentation" referenceableObject="//@sequences.0/@processings.4/@processings.0">
-        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBNZkmEd6sxtwdpecy3w"/>
+      <processings xsi:type="eef-test_1:SetAttribute" name="documentation" referenceableObject="//@sequences.0/@processings.3/@processings.5">
+        <propertiesEditionElement href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBNZkmEd6sxtwdpecy3w"/>
         <values>talk documentation</values>
-        <eContainingFeature xsi:type="ecore:EAttribute" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/documentation"/>
+        <eContainingFeature xsi:type="ecore:EAttribute" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/documentation"/>
+      </processings>
+      <processings xsi:type="eef-test_1:SetReference" name="presenter" referenceableObject="//@sequences.0/@processings.3/@processings.5" values="//@sequences.0/@processings.3/@processings.3">
+        <propertiesEditionElement href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBM5kmEd6sxtwdpecy3w"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/presenter"/>
       </processings>
-      <processings xsi:type="eef-test_1:SetReference" name="presenter" referenceableObject="//@sequences.0/@processings.4/@processings.0" values="//@sequences.0/@processings.3/@processings.2">
-        <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBM5kmEd6sxtwdpecy3w"/>
-        <eContainingFeature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/presenter"/>
+      <processings xsi:type="eef-test_1:SetReference" name="creator" referenceableObject="//@sequences.0/@processings.3/@processings.5" values="//@sequences.0/@processings.3/@processings.3">
+        <propertiesEditionElement href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBNJkmEd6sxtwdpecy3w"/>
+        <eContainingFeature xsi:type="ecore:EReference" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/creator"/>
       </processings>
-      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.4/@processings.0">
-        <processings xsi:type="eef-test_1:UnsetReference" name="presenter" referenceableObject="//@sequences.0/@processings.4/@processings.0" values="//@sequences.0/@processings.3/@processings.2">
-          <propertiesEditionElement href="platform:/plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBM5kmEd6sxtwdpecy3w"/>
-          <feature xsi:type="ecore:EReference" href="platform:/plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/presenter"/>
+      <processings xsi:type="eef-test_1:Check"/>
+      <processings xsi:type="eef-mbot:Wizard" referenceableObject="//@sequences.0/@processings.3/@processings.5">
+        <processings xsi:type="eef-test_1:UnsetReference" name="creator" referenceableObject="//@sequences.0/@processings.3/@processings.5" values="//@sequences.0/@processings.3/@processings.3">
+          <propertiesEditionElement href="../../../../../plugin/org.eclipse.emf.examples.eef.edit/models/conference.components#_KvJBNJkmEd6sxtwdpecy3w"/>
+          <feature xsi:type="ecore:EReference" href="../../../../../plugin/org.eclipse.emf.examples.eef/model/conference.ecore#//Talk/creator"/>
         </processings>
       </processings>
     </processings>
diff --git a/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/AllTests.java b/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/AllTests.java
index e8e3d19..56b1ee7 100644
--- a/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/AllTests.java
+++ b/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/AllTests.java
@@ -88,6 +88,7 @@ import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardM
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotCancelSetReferenceEOFCV;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotEditSetReferenceAdvRefTable;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotEditSetReferenceEOFCV;
+import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotRedoAddAdvanceTableComposition;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotRedoSetAttributeCheckbox;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotRedoSetAttributeEMFComboViewer;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotRedoSetAttributeMVE;
@@ -103,6 +104,7 @@ import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardM
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotSetAttributeTextArea;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotSetReferenceAdvRefTable;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotSetReferenceEOFCV;
+import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotUndoAddAdvanceTableComposition;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotUndoSetAttributeCheckbox;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotUndoSetAttributeEMFComboViewer;
 import org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference.WizardMBotUndoSetAttributeMVE;
@@ -212,6 +214,7 @@ public class AllTests extends TestCase {
 		suite.addTestSuite(WizardMBotCancelSetReferenceEOFCV.class);
 		suite.addTestSuite(WizardMBotEditSetReferenceAdvRefTable.class);
 		suite.addTestSuite(WizardMBotEditSetReferenceEOFCV.class);
+		suite.addTestSuite(WizardMBotRedoAddAdvanceTableComposition.class);
 		suite.addTestSuite(WizardMBotRedoSetAttributeCheckbox.class);
 		suite.addTestSuite(WizardMBotRedoSetAttributeEMFComboViewer.class);
 		suite.addTestSuite(WizardMBotRedoSetAttributeMVE.class);
@@ -227,6 +230,7 @@ public class AllTests extends TestCase {
 		suite.addTestSuite(WizardMBotSetAttributeTextArea.class);
 		suite.addTestSuite(WizardMBotSetReferenceAdvRefTable.class);
 		suite.addTestSuite(WizardMBotSetReferenceEOFCV.class);
+		suite.addTestSuite(WizardMBotUndoAddAdvanceTableComposition.class);
 		suite.addTestSuite(WizardMBotUndoSetAttributeCheckbox.class);
 		suite.addTestSuite(WizardMBotUndoSetAttributeEMFComboViewer.class);
 		suite.addTestSuite(WizardMBotUndoSetAttributeMVE.class);
diff --git a/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/modelingbot/wizard/conference/WizardMBotRedoAddAdvanceTableComposition.java b/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/modelingbot/wizard/conference/WizardMBotRedoAddAdvanceTableComposition.java
new file mode 100644
index 0000000..b2c1bf2
--- /dev/null
+++ b/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/modelingbot/wizard/conference/WizardMBotRedoAddAdvanceTableComposition.java
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2012 Obeo.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference;
+
+/**
+ * @author <a href="mailto:riadh.hajamor@obeo.fr">Riadh HAJ AMOR</a>
+ */
+public class WizardMBotRedoAddAdvanceTableComposition extends org.eclipse.emf.eef.modelingBot.testcase.AbstractComposedModelingBotTestCase {
+
+	public void testModelingBot() throws Exception {
+		bot.runModelingBot("org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/redoAddAdvTableCompo.modelingbot");
+	}
+
+}
diff --git a/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/modelingbot/wizard/conference/WizardMBotUndoAddAdvanceTableComposition.java b/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/modelingbot/wizard/conference/WizardMBotUndoAddAdvanceTableComposition.java
new file mode 100644
index 0000000..ec4725a
--- /dev/null
+++ b/tests/org.eclipse.emf.examples.eef.tests.junit/src/org/eclipse/emf/samples/tests/junit/modelingbot/wizard/conference/WizardMBotUndoAddAdvanceTableComposition.java
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2012 Obeo.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.emf.samples.tests.junit.modelingbot.wizard.conference;
+
+/**
+ * @author <a href="mailto:riadh.hajamor@obeo.fr">Riadh HAJ AMOR</a>
+ */
+public class WizardMBotUndoAddAdvanceTableComposition extends org.eclipse.emf.eef.modelingBot.testcase.AbstractComposedModelingBotTestCase {
+
+	public void testModelingBot() throws Exception {
+		bot.runModelingBot("org.eclipse.emf.examples.eef.tests.junit/models/modelingbot/wizard_conference/undoAddAdvTableCompo.modelingbot");
+	}
+
+}
diff --git a/tests/org.eclipse.emf.examples.eef/model/conference.genmodel b/tests/org.eclipse.emf.examples.eef/model/conference.genmodel
index 82f5b3b..5faf1d3 100644
--- a/tests/org.eclipse.emf.examples.eef/model/conference.genmodel
+++ b/tests/org.eclipse.emf.examples.eef/model/conference.genmodel
@@ -79,4 +79,4 @@
       <genFeatures createChild="false" ecoreFeature="ecore:EAttribute conference.ecore#//Room/capacity"/>
     </genClasses>
   </genPackages>
-</genmodel:GenModel>
\ No newline at end of file
+</genmodel:GenModel>
